modnet/models.py

```diff
@@ -328,29 +329,11 @@ def fit_preset(
 
         """
 
-        rlr = keras.callbacks.ReduceLROnPlateau(
-            monitor="loss",
-            factor=0.5,
-            patience=20,
-            verbose=verbose,
-            mode="auto",
-            min_delta=0,
-        )
-        es = keras.callbacks.EarlyStopping(
-            monitor="loss",
-            min_delta=0.001,
-            patience=300,
-            verbose=verbose,
-            mode="auto",
-            baseline=None,
-            restore_best_weights=True,
-        )
-        callbacks = [rlr, es]
```

Comment:
What was the reasoning for removing these?

Reply:
After some research, learning rate scheduling is something meant for SGD optimizers. The Adam optimizer automatically reduces the learning rate dynamically, separately for each weight. So seems to be actually not necessary when using Adam. Removing early stopping was more for simplicity/better tuning (relying on epochs hyperparam instead on the callback that could be a bit arbitrary in my opinion).